op-defender/psp_executor/api_test.go

```diff
@@ -6,18 +6,19 @@ import (
 	"crypto/ecdsa"
 	"crypto/rand"
 	"encoding/hex"
+	"math/big"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
 	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
 	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/gorilla/mux"
-	"math/big"
```

Comment:
The order of the imported package are important: https://golangci-lint.run/usage/linters/#gci

Reply:
Nice catch!